harper-core/src/linting/how_to.rs

```diff
@@ -0,0 +1,191 @@
+use crate::{
+    Token, TokenStringExt,
+    linting::{Lint, LintKind, PatternLinter, Suggestion},
+    patterns::{Pattern, SequencePattern},
+};
+
+pub struct HowTo {
+    pattern: Box<dyn Pattern>,
+}
+
+impl Default for HowTo {
+    fn default() -> Self {
+        let pattern = SequencePattern::default()
+            .t_aco("how")
+            .then_whitespace()
+            .then_verb();
```

Comment:
Wouldn't that also match sentences like:

- How are you?
- I like how calm you are.
- How will you make up for your mistakes?
- I will tell about how leaving my husband led to my dog winning a Nobel Prize.